UsageDocs/Installing SwiftDiscord for iOS.md

```diff
@@ -4,9 +4,9 @@
 
 ## Prerequisites
 
-* SwiftDiscord **4**
-* Xcode **8.3.3**
-* iOS SDK **10.3**
+* SwiftDiscord **5**
```

Comment:
Actually for Xcode 9 and Swift 4 you'll probably want 6.

Reply:
Hmm I got it working just fine with 5... Will revisit this when I get a chance.

Reply:
@nuclearace @tellowkrinkle Sorry I am new to this: I already have my project setup and working just fine. Is there an easy way for me to merely update the library with the Swift Package manager? Or is it best to recreate the project each time I need to update?

Reply:
Things have generally worked fine for me if I just `swift package update` and rerun `swift package generate-xcodeproj`

Reply:
That's a good question. I've never tried updating the project. Just recreating it. Maybe you could add it to version control and try to merge in the updated changes?

@tellowkrinkle The issue is with iOS targets. They aren't supported by the SPM.

Reply:
Ohh I see.
For the update to Swift 4, I would recommend recreating the project, but for smaller updates, you could try editing the existing one.  I think the only thing that would break it is if something had a new file added, in which case you would have to manually add the file to the project.

Reply:
Gotcha, but I will have to recreate my iOS target after doing that, correct?